core/src/main/java/fr/sncf/osrd/stdcm/graph/STDCMPostProcessing.java

```diff
@@ -46,7 +49,7 @@ STDCMResult makeResult(
         var physicsPath = EnvelopeTrainPath.from(infra, trainPath);
         var mergedEnvelopes = STDCMUtils.mergeEnvelopeRanges(ranges);
         var departureTime = computeDepartureTime(ranges, startTime);
-        var stops = makeStops(ranges);
+        var stops = makePathStops(ranges, infra, trainPath);
```

Comment:
This changes how the standard allowance is computed, the allowance ranges don't span across stops.

In this module all stops are considered to be places the train actually stops at, the operational points shouldn't be involved